contracts/root/predicates/ERC721Predicate.sol

```diff
@@ -199,12 +176,13 @@ contract ERC721Predicate is IErcPredicate {
     return ageOfChallengeTx > age;
   }
 
-  // function onFinalizeExit(address exitor, address token, uint256 tokenId)
-  //   external
-  //   onlyWithdrawManager
-  // {
-  //   depositManager.transferAssets(token, exitor, tokenId);
-  // }
+  function onFinalizeExit(bytes calldata data)
```

Comment:
this method is duplicated 3-4 times why not keep it in `Ipredicate` or `MixinPredicate`

Reply:
Moved to `PredicateUtils` - since other predicates already inherit from it.